portal-web/test/functional/com/liferay/portalweb/tests/AddBlogInPageLevel.testcase

```diff
@@ -0,0 +1,42 @@
+definition {
```

Comment:
The commit message should add context to what the commit is doing.

ie: `LRQA-61956 - Add test to add blog via portlet`